src/platforms/javascript/common/sourcemaps/index.mdx

```diff
@@ -38,7 +38,21 @@ $ npm install --save-dev @sentry/webpack-plugin
 $ yarn add --dev @sentry/webpack-plugin
 ```
 
-To allow the plugin to upload source maps automatically, create a `.sentryclirc` or configure environment variables as described in the [CLI configuration docs](/cli/configuration/). Then, add the plugin to your `webpack.config.js`:
+Next you will need to generate an access token for our API. Within your organization's settings navigate to Developer Settings and [create a new internal integration](/product/integrations/integration-platform/#internal-integrations). You can fill in whatever you want for the name, but it's important that you select **Releases -> Admin** for Permissions.
```

Comment:
```suggestion
```suggestion
Next you need to generate an access token for our API. Within your organization's settings, navigate to Developer Settings, [create a new internal integration](https://docs.sentry.io/product/integrations/integration-platform/#internal-integrations), and provide a name appropriate to your organization. **Important:** Select **Releases -> Admin** for Permissions.
```
```